oat-sa/tao-extension-release
# Feature/trn 605/non interactive mode

Related to https://oat-sa.atlassian.net/browse/TRN-605
 
 - Allow running the release tool in non-interactive mode : 
   - the github token can be read from env variables
   - the tool can now run in interactive  mode (by default except if the shell doesn't support inputs) or non-interactive mode. The tool can run in non-interactive mode if the flag `--no-interactive` is set. 
   - the translation flag is false by default 
 - Some dead code has been removed
 - Added the flag `--no-write` to prevent writing to the file system (could be useful if used in CI)

How to test : 
 - run the unit tests
 - try releasing the extension [taoFooBar](https://github.com/oat-sa/extension-tao-foobar) or the library [library-foobar](https://github.com/oat-sa/library-foobar/) library in interactive and non interactive modes (I let you play yourself with the options, don't hesitate to check the help `taoRelease extensionRelease --help`). In non interactive mode you shouldn't have any prompt.
